README.md

```diff
@@ -22,13 +22,13 @@ Used by the [Dart Package site](https://pub.dartlang.org/).
 You can specify either a package (+ version) or a local directory to analyze:
 
 ```
-Usage: pana [<options>] <package> [<version>]
-       pana [<options>] --source path <directory>
+Usage: pana [<options>] <published package name> [<version>]
+       pana [<options>] --source path <local directory>
 
 Options:
       --flutter-sdk     The directory of the Flutter SDK.
   -j, --json            Output log items as JSON.
-  -s, --source          The source used to find the package.
+  -s, --source          The source where the package is located (hosted on http://pub.dartlang.org, or local directory path).
                         [hosted (default), path]
   
       --hosted-url      The server that hosts <package>.
```

Comment:
Possibly mention that this overrides the default http://pub.dartlang.org.

Reply:
I think the line below is a good indication for that (defaults to https://pub.dartlang.org, and if you set to any other value, then you are on your own).